test/utils/pickers/openPicker.ts

```diff
@@ -18,11 +18,12 @@ export type OpenPickerParams =
     };
 
 export const openPicker = (params: OpenPickerParams) => {
+  const isRangePicker = params.type === 'date-range' || params.type === 'date-time-range';
```

Comment:
nit, but to be consistent:

```suggestion
  const isRangeType = params.type === 'date-range' || params.type === 'date-time-range';
```

Reply:
Kind of made sense - updated. 👌